packages/shorebird_cli/lib/src/shorebird_env.dart

```diff
@@ -238,4 +240,35 @@ class ShorebirdEnv {
       // https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml
       ||
       platform.environment.containsKey('TF_BUILD');
+
+  /// Adds shorebird.yaml to the assets section of the pubspec.yaml file.
+  void addShorebirdYamlToPubspecAssets() {
```

Comment:
Minor but this doesn't feel like it belongs in `ShorebirdEnv`. I think it would make more sense to split this out into it's own class (e.g `PubspecEditor`) since `ShorebirdEnv` should mainly be a way to query information about the current shorebird environment. This is a mutation operation which can depend on the `ShorebirdEnv` but doesn't really belong on `ShorebirdEnv` imo.

Reply:
Refactored 👍